modules/lang-painless/src/yamlRestTest/resources/rest-api-spec/test/painless/190_tsdb_downsampling.yml

```diff
@@ -0,0 +1,199 @@
+---
+"Unable to create a time series index with @timestamp runtime field":
+  - skip:
+      version: " - 8.4.99"
+      reason: "downsample introduced in 8.5.0"
+
+  - do:
+      catch: '/docvalues not found for index sort field:\[\@timestamp\]/'
```

Comment:
This is happening because for a `time_series` index we setup sorting by default on the `timestamp` field. As a result, at some point when setting up sorting on that field we try to get the Fielddata which is not there for a runtime field.

Should we check if the field is a runtime filed and return an error saying "Unable to sort using runtime field <field_name>"?